lib/savvy/index.js

```diff
@@ -10,14 +10,17 @@ mage.core.config.setTopLevelDefault('savvy', path.join(__dirname, 'config.yaml')
 var ROUTE_STR = '/savvy';
 var ROUTE_RE = new RegExp('^' + ROUTE_STR + '/');
 
-var SOCK_PATH;
 var SOCK_FILE = mage.core.config.get(['savvy', 'sockfile']);
-if (process.platform === 'win32') {
-	SOCK_PATH = path.join('\\\\.\\pipe', process.cwd(), SOCK_FILE);
-} else {
-	SOCK_PATH = path.join(process.cwd(), SOCK_FILE + '.sock');
+
+if (!path.isAbsolute(SOCK_FILE)) {
+	SOCK_FILE = path.resolve(process.cwd(), SOCK_FILE);
```

Comment:
I think `process.cwd()` is redundant here. `path.resolve()` will always return an absolute path https://nodejs.org/api/path.html#path_path_resolve_paths. Also due to its behavior, you could avoid using the `path.isAbsolute()` check.